requirements.txt

```diff
@@ -25,18 +21,18 @@ blinker==1.6.2
     #   flask
     #   gds-metrics
     #   sentry-sdk
-boto3[crt]==1.34.100
+boto3==1.34.100
```

Comment:
The `[crt]` here refers to these optional dependencies (`bototcore[crt]`):
https://github.com/boto/boto3/blob/2c3b93f41f145a828e999bd9da748ba4c2ef282c/setup.cfg#L11

`bototcore[crt]` itself refers to this optional dependency (`awscrt`):
https://github.com/boto/botocore/blob/1ad32855c799456250b44c2762cacd67f5647a6e/setup.cfg#L12

`awscrt` is already baked into this requirements file on line 13, so what uv is doing here by removing the brackets is fine.

Reply:
yeah i did a lil digging myself yesterday

https://github.com/astral-sh/uv/issues/1595

choice quotes:

>  packages don't have access to what extras they were installed only, and the only thing they see is "is package X in my extras installed".

looks like pip-tools might be moving that way too anyway